js/ui/map.js

```diff
@@ -1225,18 +1225,24 @@ function removeNode(node) {
  * mapboxgl.accessToken = myAccessToken;
  */
 
- /**
-  * Returns a Boolean indicating whether the browser [supports Mapbox GL JS](https://www.mapbox.com/help/mapbox-browser-support/#mapbox-gl-js).
-  *
-  * @function supported
-  * @param {Object} options
-  * @param {boolean} [options.failIfMajorPerformanceCaveat=false] If `true`,
-  *   the function will return `false` if the performance of Mapbox GL JS would
-  *   be dramatically worse than expected (i.e. a software renderer would be used).
-  * @return {boolean}
-  * @example
-  * mapboxgl.supported() // = true
-  */
+/**
+ * Gets the version number of Mapbox GL JS.
+ *
+ * @var version
+ */
```

Comment:
I do not know of a way to declare that a `var` is a string. If I add a type, e.g. `@var version {string}`, compilation for that item seems to fail. Do you know of another way this should be marked with JSDoc?

Regarding context, what do you think of this? ` To learn more about the differences between published versions of Mapbox GL JS, have a look at [the changelog](https://github.com/mapbox/mapbox-gl-js/blob/master/CHANGELOG.md).` Sufficient? I guess I'm not sure what else to say here ... open to ideas.